tests/unittest/inputs/test_registry_mixed_multimodal.py

```
@@ -0,0 +1,702 @@
+        # Build a realistic expanded prompt: each item's placeholder becomes the
+        # matching number of that modality's feature tokens, in the resolved
+        # (interleaved) prompt order. This lets the registry's mask-based
+        # re-bake of `multimodal_embedding_lengths` count real embed slots.
+        self.lengths_seen_by_expander = list(num_mm_tokens_per_placeholder)
+        assert mm_data is not None
+        feature_id = {"image": _IMAGE_FEATURE_ID, "audio": _AUDIO_FEATURE_ID}
+        # `item_order` for this prompt is image, audio, image (see
+        # `get_mm_item_order`); zip it with the resolved per-item counts.
+        order = self.get_mm_item_order(prompt_token_ids, mm_data)
+        expanded = [10]
+        for (modality, _), count in zip(order, num_mm_tokens_per_placeholder):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)